notebooks/notebooks/iclr2022/utils.py

```diff
@@ -0,0 +1,60 @@
+"""Utility functions for plot notebooks."""
+import pathlib
+from typing import Iterable, List, TypeVar
+
+import pandas as pd
+
+from sgf_parser import game_info
+
+
+T = TypeVar("T")
+
+
+def flatten_2d_list(lists: Iterable[List[T]]) -> List[T]:
+    """Flattens a 2D list into a 1D list."""
+    return sum(lists, [])
+
+
+def get_style(style_name: str) -> str:
+    """Gets Matplotlib style sheet with a given name."""
+    return f"./matplotlib-style-sheets/{style_name}.mlpstyle"
```

Comment:
Depending on CWD is a bit fragile, consider making it relative to the script path https://note.nkmk.me/en/python-script-file-path/ or https://docs.python.org/3/library/importlib.resources.html#module-importlib.resources